modules/nf-core/atlasgeneannotationmanipulation/gtf2featureannotation/main.nf

```diff
@@ -0,0 +1,43 @@
+process ATLASGENEANNOTATIONMANIPULATION_GTF2FEATUREANNOTATION {
+    tag '$gtf'
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::atlas-gene-annotation-manipulation=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/atlas-gene-annotation-manipulation%3A1.1.0--hdfd78af_0':
+        'quay.io/biocontainers/atlas-gene-annotation-manipulation:1.1.0--hdfd78af_0' }"
+
+    input:
+    val meta
+    path gtf
```

Comment:
I'd say its better to pack meta and gtf as a tuple [like this](https://github.com/nf-core/modules/blob/master/modules/nf-core/abacas/main.nf) because that's the default output structure in modules. That way if I want to pass an output from another process to this one, I can pass it as it is. Otherwise I would need to manipulate the channel to create a separate meta and gtf channels which is unnecessary.